odoo_module_migrate/migration_scripts/migrate_120_130.py

```diff
@@ -2,6 +2,22 @@
 # @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
+_TEXT_WARNINGS = {
+    ".xml": {
+        r".*active.*toggle_button":
+            'You can remove actions archive/unarchive',
+        r".*active.*boolean_button":
+            'You can remove actions archive/unarchive'
+    },
+    ".py": {
+        r".*oldname":
+            "oldname is not supported yet. Create a migration script",
+        r"Float.*digits.*precision":
```

Comment:
Maybe more strict, e.g.

        r"fields.Float\(.*digits=[^, )]*get_precision":

or for case of multi-line field definition:


        r"digits=[^, )]*get_precision":

Reply:
Changed in new push